core/storage/ipfs/ipfs_datastore_error.hpp

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_STORAGE_IPFS_DATASTORE_ERROR_HPP
+#define CPP_FILECOIN_CORE_STORAGE_IPFS_DATASTORE_ERROR_HPP
+
+#include "common/outcome.hpp"
+
+namespace fc::storage::ipfs {
+
+  /**
+   * @brief Type of errors returned by Keystore
+   */
+  enum class IpfsDatastoreError {
+    NOT_FOUND = 1,
+    CANNOT_STORE = 2,
+
+    UNKNOWN = 1000
```

Comment:
```suggestion
    UNKNOWN
```
this one is redundant as well

Reply:
I would leave 1000 in case of new errors added.

Reply:
There is no need to keep error codes values.